app/actions/handlers.py

```diff
@@ -155,11 +151,25 @@ async def action_pull_events(integration: Integration, action_config: PullEvents
                 for feature in geostore.attributes.geojson["features"]
             ]
         )
-        for partition in utils.generate_geometry_fragments(geometry_collection=geometry_collection):
+        try:
+            for partition in utils.generate_geometry_fragments(geometry_collection=geometry_collection):
+                geostore = await dataapi.create_geostore(geometry=mapping(partition))
+
+                await state_manager.add_geostore_id(aoi_data.id, geostore.gfw_geostore_id)
+
+        except ValueError:
```

Comment:
Is this catching an exception from calling `generate_geometry_fragments` or from `create_geostore`?

What I would like here is to let this for-loop complete, but keep track of errors reported from create_geostore. That way we can log the one Warning message but let the execution continue.